planet/src/main/webapp/places.js

```diff
@@ -22,48 +24,60 @@ function initMap() {
         center: TORONTO_COORDINATES,
         zoom: 8
     });
-    let infoWindow = new google.maps.InfoWindow;
-        // Gets users current position
-        if (navigator.geolocation) {
-            navigator.geolocation.getCurrentPosition(function(position) {
-            let pos = {
-              lat: position.coords.latitude,
-              lng: position.coords.longitude
+
+    // Gets users current location
+    if (navigator.geolocation) {
+        navigator.geolocation.getCurrentPosition(function(position) {
+            pos = {
+            lat: position.coords.latitude,
+            lng: position.coords.longitude
             };
-            infoWindow.setPosition(pos);
-            infoWindow.setContent('You are here.');
-            infoWindow.open(map);
+            // Create draggable marker at current location
+            let searchMarker = new google.maps.Marker({
+                position: pos,
+                map: map,
+                animation: google.maps.Animation.DROP,
+                draggable: true
+            });
             map.setCenter(pos);
         }, function() {
```

Comment:
add a comment here explaining when this function is called. Otherwise, the reader needs to go find the documentation for getCurrentPosition to understand what this code is doing.